mode-apis-base/src/alertingModels.ts

```diff
@@ -163,12 +163,47 @@ export const isMetricsThresholdAlertCondition = (obj: unknown): obj is MetricsTh
 
 
 
-/**
- * TODO - Define the attributes required for AlertAction
- */
-export interface AlertAction {
+export interface AlertMessageTemplate {
+    readonly title: string;
```

Comment:
I am guessing this can be used as email `subject` if the AlertRecipientType is `email`?

Reply:
Exactly. How this data is used depends on the recipient type.